arviz/stats/diagnostics.py

```diff
@@ -996,3 +996,137 @@ def _multichain_statistics(ary, focus="mean"):
         ess_tail_value,
         rhat_value,
     )
+
+
+def psens(data, *, component, var_names=None, delta=0.01, dask_kwargs=None):
+    """Compute power-scaling sensitivity diagnostic.
+
+    Power-scales the prior or likelihood and calculates how much the posterior is affected.
+
+    Parameters
+    ----------
```

Comment:
the docstring for the `delta` argument is missing. The order of the arguments `component` and `var_names` has to be switched.